brain/python/numpyArrray.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
When I included the numpy header (before moving the code to arrayHelpers), I got a bunch of warnings that you disabled by some defines. That is why I moved all that in a single header which is only included from arrayHelpers.cpp but that can be reused later.